src/types/futures.ts

```diff
@@ -851,14 +852,14 @@ export interface ModifyOrderParams {
   quantity: string;
   price: string;
   priceMatch?:
-    | 'OPPONENT'
```

Comment:
Formatted by the language server...

Reply:
This was automatically changed by your editor? I have eslint configured to run on-save, but you can run it manually on only this file using this command:
```
npx eslint --fix src/types/futures.ts
```

That should correct the formatting here